include/ocpp/common/websocket/websocket.hpp

```diff
@@ -9,6 +9,10 @@
 #include <ocpp/common/websocket/websocket_tls.hpp>
```

Comment:
Now that we have abstracted, do we still require these headers? The purpose of the refactor was to remove any dependency on them.

Reply:
Well I only changed the interface, I think you are right.